src/python/pants/core/goals/fmt.py

```diff
@@ -135,6 +136,11 @@ def register_options(cls, register) -> None:
             advanced=True,
             type=bool,
             default=False,
+            removal_version="2.11.0.dev0",
+            removal_hint=(
+                "Formatters are now broken into multiple batches by default using the "
+                "`--batch-size` argument."
```

Comment:
Might be helpful to add this:

> \n\nIf you'd like to keep this option's behavior, set `[fmt].batch_size = 1`. However, you'll likely get better performance by using a larger batch size because of less overhead launching processes.